cni/network/network_linux.go

```diff
@@ -33,6 +37,13 @@ func addDefaultRoute(gwIPString string, epInfo *network.EndpointInfo, result *cn
 	result.Routes = append(result.Routes, &cniTypes.Route{Dst: dstIP, GW: gwIP})
 }
 
+func addSnatForDNS(gwIPString string, epInfo *network.EndpointInfo, result *cniTypesCurr.Result) {
+	_, dnsIPNet, _ := net.ParseCIDR("168.63.129.16/32")
```

Comment:
move this address as constant

Reply:
this is the only place it's used, and it's also a temporary change